lib/hivex-internal.h

```diff
@@ -347,7 +348,7 @@ extern int _hivex_get_values (hive_h *h, hive_node_h node, hive_value_h **values
 
 /* These limits are in place to stop really stupid stuff and/or exploits. */
 #define HIVEX_MAX_SUBKEYS      200000
-#define HIVEX_MAX_VALUES       100000
+#define HIVEX_MAX_VALUES        55000
```

Comment:
Dont change this, keep the old value

Reply:
Are you sure? The change was merged in from your defragment branch.